docs/docs/reference/project-files/dashboards.md

```diff
@@ -57,3 +63,6 @@ _**`security`**_ - define a [security policy](../../develop/security) for the da
   - _**`include`**_ - List of dimension or measure names to include in the dashboard. If `include` is defined all other dimensions and measures are excluded. _(optional)_
     - **`if`** - Expression to decide if the column should be included or not. It can leverage templated user attributes. Needs to be a valid SQL expression that evaluates to a boolean. _(required)_
     - **`names`** - List of fields to include. Should match the `name` of one of the dashboard's dimensions or measures. _(required)_
+
+**`kind`**
+ — Can be one of `{source,model,dashboard}` and used by Rill to identify the resource kind. This property is optional and only necessary when the YAML file exists outside of the default location (e.g. a dashboard YAML outside of the `dashboards` directory).
```

Comment:
> Can be one of `{source,model,dashboard}`

Since this page is specific to dashboards may be we should specify `kind` as `dashboard` only and similarly on sources page specify `kind` as `source` only ?

Reply:
Good callout. Since we don't currently exactly have an equivalent page for models, I wanted to at least point out what the available options are. I've rewritten this section to make it more specific though to the resource.

Reply:
Sure. By the way there is one for both source and model as well :`docs/docs/reference/project-files/models.md`